src/extension.ts

```diff
@@ -96,10 +98,13 @@ export async function activateInternal(context: vscode.ExtensionContext) {
 
     await callWithTelemetryAndErrorHandling('azureApiManagement.Activate', async (activateContext: IActionContext) => {
         activateContext.telemetry.properties.isActivationEvent = 'true';
-        ext.azureAccountTreeItem = new AzureAccountTreeItem();
-        context.subscriptions.push(ext.azureAccountTreeItem);
-
-        ext.tree = new AzExtTreeDataProvider(ext.azureAccountTreeItem, 'azureApiManagement.LoadMore');
+        AzureSessionProviderHelper.activateAzureSessionProvider(context);
+        const sessionProvider = AzureSessionProviderHelper.getSessionProvider();
+        const azureAccountTreeItem = createAzureAccountTreeItem(sessionProvider);
+        context.subscriptions.push(azureAccountTreeItem);
+        ext.azureAccountTreeItem = azureAccountTreeItem;
+        
+        ext.tree = new AzExtTreeDataProvider(azureAccountTreeItem, 'azureApiManagement.loadMore');
```

Comment:
`azureApiManagement.LoadMore` is changed to `azureApiManagement.loadMore`, is it on purpose?

Reply:
updated